Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/Result.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Frends.Kafka.Produce.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Message sent.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// Persistence status.
```

Comment:
Any meaningful examples we can give?

Reply:
I looked at Kafka lib and it seems that it only has three possibilities - https://docs.confluent.io/platform/current/clients/confluent-kafka-dotnet/_site/api/Confluent.Kafka.PersistenceStatus.html. Can we enumerate all those in summary? This would clear up any ambiguousness for anyone thinking "what can this field be".